tests/e2e/fixtures/cluster.py

```diff
@@ -67,6 +67,16 @@ def create_iam_service_account(
     retcode = subprocess.call(cmd)
     assert retcode == 0
 
+
+def get_iam_service_account_roleArn(service_account_name, namespace):
```

Comment:
I am fine with this but it might also not be required since the load balancer chart has option of setting service account creation to false